### DISCUSSION TITLE
[0.7.0] - 2023-05-11

### DISCUSSION THREAD

**Discussion by SkalskiP** [roboflow/supervision#95] [Announcements]
## 🚀 Added

- `Detections.from_yolo_nas` to enable seamless integration with [YOLO-NAS](https://github.com/Deci-AI/super-gradients/blob/master/YOLONAS.md) model. (https://github.com/roboflow/supervision/pull/91)
- Ability to load datasets in YOLO format using `Dataset.from_yolo`. (https://github.com/roboflow/supervision/pull/86)
- `Detections.merge` to merge multiple `Detections` objects together. (https://github.com/roboflow/supervision/pull/84)

## 🌱 Changed

- `LineZoneAnnotator.annotate` to allow for the custom text for the in and out tags. (https://github.com/roboflow/supervision/pull/44)

## 🛠️ Fixed

- `LineZoneAnnotator.annotate` does not return annotated frame. (https://github.com/roboflow/supervision/pull/81)

## 🏆 Contributors

- @SkalskiP 
- @iPoe
- @hardikdava

<hr /><em>This discussion was created from the release <a href='https://github.com/roboflow/supervision/releases/tag/0.7.0'>[0.7.0] - 2023-05-11</a>.</em>

**Comment by ghost** [comment]
funny
